src/app/globals.css

```
@@ -11,61 +11,86 @@
-  max-width: 100%;
-  overflow-x: hidden;
 }
 
-@media (max-width: 600px) {
-  body {
-    width: 100%;
-    padding: 0;
+@layer base {
+  html {
+    @apply font-sans w-full bg-surface text-content leading-normal antialiased;
+    font-size: 14px;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)